docs/ops/comparison/IsNaN.md

```diff
@@ -0,0 +1,49 @@
+# IsNaN {#openvino_docs_ops_comparison_IsNaN_10}
+
+**Versioned name**: *IsNaN-10*
+
+**Category**: *Comparison*
+
+**Short description**: *IsNaN* returns boolean mask of a given tensor which mapps `NaN` to `True`.
+
+**Detailed description**: *IsNaN* returns the bolean mask of the input tensor in which `True` corresponds to `NaN` and `False` to other values.
+* The output tensor has the same shape as input tensor.
+* The `i`'th element of the output tensor is `True` if  `i`'th element of the input tensor is `NaN`. Otherwise it is `False`.
+* For example, for given input tensor `[NaN, 2.1, 3.7, NaN]` the output tensor is `[True, False, False, True]`.
```

Comment:
Thanks for the clear example! How is infinity handled?

Reply:
Infinity is handled the same way as in numpy - it is mapped to False.

Reply:
Thanks, could you add that to the example code or description?

Reply:
I have added `Inf` case to the example.